src/styles/dropdown.ts

```diff
@@ -13,7 +13,7 @@ export const dropdownCss = css`
     min-width: 100%;
     overflow: hidden;
     position: absolute;
-    z-index: 1;
+    z-index: 1000;
```

Comment:
Well that's certainly future-proof...

Reply:
the max is apparently 2147483647 but i figure this is enough that if you make it higher than you really want it above